lib/ramble/ramble/application.py

```diff
@@ -72,6 +72,23 @@ def _get_context_display_name(context):
     )
 
 
+def _check_shell_support(app_inst):
```

Comment:
Should this also apply to modifiers and package managers?

Either in their base classes, or in the application class where the instances for the other objects are created?